samples/c_cxx/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.13)
+
+# Project
+project(onnxruntime_samples C CXX)
+if (WIN32)
+ string(APPEND CMAKE_CXX_FLAGS " /W4")
+else()
+ string(APPEND CMAKE_CXX_FLAGS " -Wall -Wextra")
+ string(APPEND CMAKE_C_FLAGS " -Wall -Wextra")
+endif()
+
+message("${CMAKE_SYSTEM_INCLUDE_PATH}")
+find_package(JPEG REQUIRED)
+find_package(PNG)
+
```

Comment:
Is the JPEG package automatically installed? Should we include the build command as part of a README.md? It should be automatic to get started with the  examples once ORT is build (e.g. similar to MNIST).

```
CMake Error at D:/cmake3.13.1/share/cmake-3.13/Modules/FindPackageHandleStandardArgs.cmake:137 (message):
  Could NOT find JPEG (missing: JPEG_LIBRARY JPEG_INCLUDE_DIR)
```

Reply:
> Is the JPEG package automatically installed

No. People need to manually install it.
Now I changed it as optional, so it won't be an issue anymore.